.env.onprem.example

```
@@ -0,0 +1,81 @@
+# ---------------------------------------------------------------------------
+# Generate with: openssl rand -hex 32
+ENCRYPTION_KEY=generate_a_64_char_hex_string_here
+SESSION_SECRET=generate_another_64_char_hex_string_here
+CSRF_SECRET=generate_another_64_char_hex_string_here
+CRON_SECRET=generate_another_64_char_hex_string_here
+
+# ---------------------------------------------------------------------------
+# Application URLs (REQUIRED)
+# ---------------------------------------------------------------------------
+NEXT_PUBLIC_APP_URL=https://pagespace.clinic.local
+NEXTAUTH_URL=https://pagespace.clinic.local
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXTAUTH_URL key should go before the NEXT_PUBLIC_APP_URL key

(UnorderedKey)